packages/build-config/package.json

```diff
@@ -0,0 +1,88 @@
+{
+	"name": "@wordpress/build-config",
```

Comment:
I would personally put it inside `@wordpress/webpack-config` to make it clear that it's specific to this tool.

Reply:
Updated acc77a4c8e3d7d3511d79e2435c88d1f2a5b1cb9